Description: Upstream changes introduced in version 4.10.0-0ubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 unity (4.10.0-0ubuntu1) oneiric; urgency=low
 .
   * New upstream release:
     - unity task switcher (alt-tab, alt-down) shows empty spaces for iconified
       windows in window detail mode (lp: #828348)
     - compiz crashed with SIGSEGV in
       unity::DeviceLauncherIcon::OnStopDriveReady() (lp: #824093)
     - Dash performance issues with 'Active Blur' enabled (lp: #824890)
     - compiz crashed with SIGABRT in raise() when setting background
       to full color (lp: #829049)
     - Dragging from dash to launcher still doesn't work (lp: #824833)
     - .desktops dragged on to unity launcher will not be added to
        com.canonical.Unity.Launcher favorites unless moved in the launcher
        (lp: #830536)
     - application's menu shown when the cursor is a little left
       to window controls (lp: #820293)
     - Ubuntu Start launcher item should always start the dash with
       the Home screen (lp: #825034)
     - search spinner does not stop hence nothing can be launched with
       the Return key (lp: #824836)
   * debian/control: updated nux requirement
   * debian/rules: updated shlibs
 .
 The person named in the Author field signed this changelog entry.
Author: Sebastien Bacher <seb128@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/820293
Bug-Ubuntu: https://bugs.launchpad.net/bugs/824093
Bug-Ubuntu: https://bugs.launchpad.net/bugs/824833
Bug-Ubuntu: https://bugs.launchpad.net/bugs/824836
Bug-Ubuntu: https://bugs.launchpad.net/bugs/824890
Bug-Ubuntu: https://bugs.launchpad.net/bugs/825034
Bug-Ubuntu: https://bugs.launchpad.net/bugs/828348
Bug-Ubuntu: https://bugs.launchpad.net/bugs/829049
Bug-Ubuntu: https://bugs.launchpad.net/bugs/830536

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- unity-4.10.0.orig/services/CMakeLists.txt
+++ unity-4.10.0/services/CMakeLists.txt
@@ -39,7 +39,7 @@ set(CFLAGS
     "-DGETTEXT_PACKAGE=\"unity\""
     "-DINDICATORDIR=\"${_indicatordir}\""
     "-DINDICATORICONDIR=\"${_iconsdir}\""
-    "-Werror -Wall"
+    "-Wall"
     )
 add_definitions(${CFLAGS})
 
